test/integration/userStatus.test.js

```diff
@@ -63,6 +63,44 @@ describe("UserStatus", function () {
           return done();
         });
     });
+
+    it("Should not get the idle userStatus for archived user", async function () {
+      const archivedUserId = await addUser(userData[5]); // User with role archived true
+      await updateUserStatus(archivedUserId, generateUserStatusData("IDLE", new Date(), new Date()));
+      const response = await chai.request(app).get("/users/status?state=IDLE");
+      expect(response).to.have.status(200);
+      expect(response.body.message).to.equal("All User Status found successfully.");
+      expect(response.body.totalUserStatus).to.be.a("number");
+      expect(response.body.totalUserStatus).to.equal(0);
+      expect(response.body.allUserStatus).to.be.a("array");
+      expect(response.body.allUserStatus.length).to.equal(0);
+    });
+
+    it("Should get the idle userStatus for active user or non archived user", async function () {
+      const nonArchivedUserId = await addUser(userData[6]); // User with role archived false
+      await updateUserStatus(nonArchivedUserId, generateUserStatusData("IDLE", new Date(), new Date()));
+      const response = await chai.request(app).get("/users/status?state=IDLE");
+      expect(response).to.have.status(200);
+      expect(response.body.message).to.equal("All User Status found successfully.");
+      expect(response.body.totalUserStatus).to.be.a("number");
+      expect(response.body.totalUserStatus).to.equal(1);
+      expect(response.body.allUserStatus).to.be.a("array");
+      expect(response.body.allUserStatus.length).to.equal(1);
+    });
+
+    it("Should only get the idle userStatus only for active user when both archived and active users are there in DB", async function () {
```

Comment:
can we change the statement to `should return only non-archived user`

the above test for `should not get archived user` can be removed as this test checks for both